src/frontend/komponenter/Behandling/Aktivitet/Aktivitet/Annet.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+import { ISærligeTilsynsbehov } from '../../../../typer/overgangsstønad';
+import { GridTabellRad, GridTabell } from '../../../Felleskomponenter/Visning/StyledTabell';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
+import { PopoverOrientering } from 'nav-frontend-popover';
+import { Normaltekst, Element } from 'nav-frontend-typografi';
+import styled from 'styled-components';
+import { Søknadsgrunnlag } from '../../../Felleskomponenter/Visning/DataGrunnlagIkoner';
+import { DinSituasjonTilTekst, EDinSituasjon } from './typer';
+import { v4 as uuidv4 } from 'uuid';
+import { formaterNullableIsoDato } from '../../../../utils/formatter';
+
+const StyledList = styled.ul`
+    list-style-type: square;
+    margin: 0;
+    padding-left: 1rem;
+`;
+
+const hjelpetekst = (
+    <ul>
+        <Normaltekst>
+            Mulig alternativer i søknadsdialog:
+            <li>Jeg er syk</li>
+            <li>Barnet mitt er sykt </li>
+            <li>Jeg har søkt om barnepass, men ikke fått plass enda</li>
+            <li>
+                Jeg har barn som trenger særlig tilsyn på grunn av fysiske, psykiske eller store
+                sosiale problemer
+            </li>
+            <li>Nei</li>
+        </Normaltekst>
+    </ul>
+);
+
+const TittelHjelpetekstWrapper = styled.div`
+    display: flex;
+    .litenSpacingHoyre {
+        padding-right: 1rem;
+    }
+`;
+
+interface Props {
+    dinSituasjon: EDinSituasjon[];
+    særligTilsynsbehov: ISærligeTilsynsbehov[];
+}
+
+const Annet: FC<Props> = ({ dinSituasjon, særligTilsynsbehov }) => {
+    return (
+        <>
+            <GridTabell kolonner={3}>
+                <TittelHjelpetekstWrapper className={'førsteDataKolonne'}>
+                    <Element className={'litenSpacingHoyre'}>Annet</Element>
+                    <Hjelpetekst type={PopoverOrientering.OverVenstre}>{hjelpetekst}</Hjelpetekst>
+                </TittelHjelpetekstWrapper>
+
+                <Søknadsgrunnlag />
+                <Normaltekst className={'førsteDataKolonne'}>Mer om søkers situasjon</Normaltekst>
+                <StyledList>
+                    {dinSituasjon.map((svarsalternativ) => (
+                        <li key={svarsalternativ}>
+                            <Normaltekst>{DinSituasjonTilTekst[svarsalternativ]}</Normaltekst>
+                        </li>
+                    ))}
+                </StyledList>
+
+                <GridTabellRad kolonner={3} overTabellRadPadding={2}>
+                    {særligTilsynsbehov.map((barnetsBehov) => (
+                        <GridTabell key={uuidv4()} kolonner={3} underTabellMargin={2}>
```

Comment:
key= uuidv4 er litt scary - har vi noen id å bruke? Evt. indeks er bedre enn uuid tror jeg...

Reply:
Har hatt problemer med bruk av indeks ved mapping av samme React komponent tidligere. Men siden vi ikke skal endre data denne gang, er det kanskje ikke så farlig. 

@charliemidtlyng Hvorfor er bruk av uuidv4 scary? Tror det var det vi brukte for å løse buggy oppførsel i søknadsdialogen den gang

Reply:
Fordi du genrerer ny hver gang det kjøres render. Vi kan godt sette uuid initielt for hvert tilsynsbehov+++ etter å ha hentet data, men da må den være den samme for hver re-rendring. Hvis den er forskjellig vil React re-rendre alle objektene hver gang noe på siden de er "endret" (fått ny key). 

Index er risikofylt hvis man driver med endring/oppdatering av lister, ja. Men hvis listen mottas fra backend og kun skal vises frem er det greit nok. 

For alle situasjoner som krever endring av listen så anbefaler jeg å generere en nøkkel idét man laster data slik at man endrer basert på nøkler og ikke indekser.

Reply:
Okei! Ser den. Takk for lengthy forklaring. Fikset det i ny commit!